cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManagerTest.java

```diff
@@ -267,7 +270,7 @@ public void testMaximumActiveTasks() {
     OperationFuture future1 =
         userTaskManager.getOrCreateUserTask(mockHttpServletRequest1, mockHttpServletResponse, uuid -> future, 0, true, null).get(0);
     Assert.assertEquals(future, future1);
-
+    EasyMock.verify(mockHttpServletRequest1, mockHttpServletResponse);
```

Comment:
Should this be `EasyMock.verify(mockHttpSession1, mockHttpServletRequest1, mockHttpServletResponse);`?